wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-[4] Bottom (long) {Similar to <a href="osu!stream" class="uri" title="wikilink">osu!stream</a>}</p></td>
-</tr>
-<tr class="even">
-<td><p>rowspan = 3|Score meter type (configures accuracy meter appearing below the beatmap).</p></td>
-<td><p>None</p></td>
-</tr>
-<tr class="odd">
-<td><p>Colour [Default for Catch the Beat]</p></td>
-<td><p><img src="OP SM Colour.jpg" title="fig:OP SM Colour.jpg" alt="OP SM Colour.jpg" width="350" height="20" /><br />
-Shows a history of your hit values (based on colours of default skin's 300/100/50/Miss).</p>
-<table>
-<caption>Default colourings</caption>
-<tbody>
-<tr class="odd">
-<td></td>
-<td><p>300</p></td>
-<td></td>
-<td><p>100</p></td>
-<td></td>
-<td><p>50</p></td>
-<td></td>
-<td><p>Miss</p></td>
-<td></td>
-<td><p>Blank/Next note</p></td>
-</tr>
-</tbody>
-</table></td>
-</tr>
-<tr class="even">
-<td><p>Hit-error [Default for other modes]</p></td>
-<td><p><img src="OP SM HE.jpg" title="fig:OP SM HE.jpg" alt="OP SM HE.jpg" /><br />
-Shows precisely how accurate you were with each hit. Judgement Bar length adjusts based on OD (Accuracy). Left - Center - Right = Early - Perfect - Late</p>
-<table>
-<caption>Default colourings</caption>
-<tbody>
-<tr class="odd">
-<td></td>
-<td><p>Perfect</p></td>
-<td></td>
-<td><p>300</p></td>
-<td></td>
-<td><p>100</p></td>
-<td></td>
-<td><p>50</p></td>
-<td><p>[pointer]</p></td>
-<td><p>Average</p></td>
-</tr>
-</tbody>
-</table></td>
-</tr>
-<tr class="odd">
-<td><p>Score metre size</p></td>
-<td><p>colspan = 2|Set your score metre size.<br />
-Default is 1.00x. Size range from 0.50x - 2.00x.</p></td>
-</tr>
-<tr class="even">
-<td><p>Always show key overlay</p></td>
-<td><p>colspan = 2|Shows the key status overlay even when playing normally. Handy for streaming your play.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Show approach circle on first &quot;Hidden&quot; object.</p></td>
-<td><p>colspan = 2|Show an approach circle on the first hidden object. <a href="Media:OP_AC_on_Hidden.jpg" title="wikilink">Picture example</a>.</p></td>
-</tr>
-<tr class="even">
-<td><p>Scale osu!mania scroll speed with BPM</p></td>
-<td><p>colspan = 2|Adjust the scroll speed based on the BPM of the beatmap.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Remember osu!mania scroll speed per beatmap.</p></td>
-<td><p>colspan = 2|Remember the scroll speed you personally set per beatmap.</p></td>
-</tr>
-</tbody>
-</table>
+| Name                                          | Effect                                                                         |
+|-----------------------------------------------|--------------------------------------------------------------------------------|
+| Background Dim                                | Adjust the level of dimming applied to the background while playing a beatmap. |
+| Progress Display                              | Configure where and how the song progress bar is displayed.                    |
```

Comment:
Could note that using Pie will also display the time before the map actually starts